io-engine/src/bdev/nexus/nexus_io.rs

```diff
@@ -38,19 +44,74 @@ use crate::core::{
     Reactors,
 };
 
+#[cfg(feature = "nexus_io_tracing")]
+mod debug_nexus_io {
+    use std::sync::atomic::{AtomicU64, Ordering};
+
+    static SERIAL: AtomicU64 = AtomicU64::new(0);
+
+    pub(super) fn new_serial() -> u64 {
+        SERIAL.fetch_add(1, Ordering::SeqCst)
+    }
+}
+
+#[cfg(feature = "nexus_io_tracing")]
+macro_rules! trace_io {
+    ($($arg:tt)*) => {{ debug!($($arg)*); }}
```

Comment:
should this be trace! as per the macro name?

Reply:
yes, will fix.